### DISCUSSION TITLE
Loading data from BigQuery using Python API and Delta Rust

### DISCUSSION THREAD

**Discussion by djouallah** [delta-io/delta-rs#2058] [Show and tell]
it is an awesome experience the native arrow integration just worked
https://datamonkeysite.com/2024/01/08/using-arrow-and-delta-rust-to-transfer-data-from-bigquery-to-fabric-onelake/

**Comment by rtyler** [comment]
@djouallah Big Query can also  `EXPORT DATA` into parquet format which can be loaded into a Delta table with some clever scripting.   Passing around arrow in Python though is pretty cool, I agree :smile:
